packages/ics721/src/execute.rs

```diff
@@ -506,6 +529,19 @@ where
                     .add_attribute(
                         "cw721_base_code_id",
                         cw721_base_code_id.map_or_else(|| "none".to_string(), |or| or.to_string()),
+                    )
+                    .add_attribute(
```

Comment:
Can you explain what this line is doing?

Reply:
Here `cw721_base_code_id` is of type `Option<u64`. In case of response provides attribute telling "none" is provided - instead of provding an empty string.

Reply:
My question here is why do we have the plaintext "none" instead of an actual None?

Reply:
Attribute must be a string, that's why we need to convert all these data.